tests/jballerina-integration-test/src/test/resources/observability/tracing_tests/Dependencies.toml

```diff
@@ -7,5 +7,3 @@ version = "0.0.0"
 org = "ballerina"
 name = "observe"
 version = "0.9.0"
-
```

Comment:
Unrelated change. Let's revert this.

Reply:
[master](https://github.com/ballerina-platform/ballerina-lang/blame/master/tests/jballerina-integration-test/src/test/resources/observability/tracing_tests/Dependencies.toml) seem to have an extra line, hence keeping the change.